models/Document/Editable/Video.php

```diff
@@ -696,9 +696,15 @@ private function getDailymotionCode()
         }
 
         if ($dailymotionId) {
-            $width = $config['width'] ?? '100%';
+            $width = '100%';
+            if (array_key_exists('width', $config)) {
```

Comment:
default value should be used if null is passed in width so we should revert back to null coalescing